debian/control

```diff
@@ -3,6 +3,11 @@ Section: admin
 Priority: extra
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
 Build-Depends: debhelper (>= 9),
+               libxml2-dev,
+               libxslt-dev,
+               python-dev,
+               lib32z1-dev,
```

Comment:
You can see at commit 2745229 the build was failing. The errors in travis-ci were due to lxml not being able to install due to missing deps.